solutions/gold/usaco-993.mdx

```diff
@@ -2,14 +2,24 @@
 id: usaco-993
 source: USACO Gold 2020 January
 title: Time is Mooney
-author: Nathan Gong, Ryan Chou
+author: Nathan Gong, Ryan Chou, David Guo
 ---
 
 [Official Analysis (C++)](http://www.usaco.org/current/data/sol_time_gold_jan20.html)
 
 ## Explanation
 
-We only need to keep track of the number of days that Bessie has been traveling and the amount of mooney that she has made. For every day up to $T_{\text{max}}$, we'll try to travel to each city, and update the mooney made if the current path is more optimal. The maximum amount of money she makes is $1000t - t^2$, in the case where she makes $1000$ mooney per city and the cost is $1$. Her earnings become negative when $t > 1000$, so we only have to check her movement across the cities for at most $1000$ days.
+The problem involves maximizing the profit Bessie can earn while traveling
```

Comment:
should probably still calculate the upper bound for the # of days she travels

also, I think it would be nice if you could go a bit into the motivation for using dp

e.g., in a trip, mention that only two things matter in evaluating it: the city we are in, and the day we are on